src/scripting.md

```diff
@@ -98,514 +211,551 @@ input: setCookie  %ORIGIN% foo=bar
 Browser Support: IE
 ```markup
-usage: expireCache	<seconds>
-example: expireCache	86400
+usage:   expireCache <seconds>
+example: expireCache 86400
 
 <seconds> - Any resources with a cache lifetime less than this amount of time will be forced to expire.
 ```
 
-#### firefoxPref
+#### `firefoxPref`
 Allows you to specify arbitrary preferences that will be configured before launching the browser.
 Browser Support: Firefox
 ```markup
-usage: firefoxPref	<pref>    <value>
+usage:   firefoxPref <pref> <value>
 examples:
-firefoxPref    network.http.pipelining    false
-firefoxPref    network.http.pipelining.maxrequests    5
-firefoxPref    general.useragent.override    "Some User Agent String"
+firefoxPref network.http.pipelining false
+firefoxPref network.http.pipelining.maxrequests 5
+firefoxPref general.useragent.override "Some User Agent String"
 
 <pref> - The preference that is to be modified
 <value> - The value to use.  String values should be enclosed in quotes like the example.
 ```
 
-#### setEventName
+#### `setEventName`
 Sets the name of the event for the next measurable operation. It is important to only set this right before the action that will generate the activity you want to measure so that you don't inadvertently measure other page activity. Without explicit event names each step will be automatically named Step_1, Step_2, etc.
 Browser Support: IE
 ```markup
-usage: setEventName	<event name>
-example: setEventName	loadWebmail
+usage:   setEventName <event name>
+example: setEventName loadWebmail
 
 <event name> - Name to use for the event about to occur (in resulting log files)
 ```
 
-#### setLocation
+#### `setLocation`
 Specifies a geolocation override position.
 Browser Support: Chrome
 ```markup
-usage: setLocation	<lat>,<lng>    <accuracy>
-example: setLocation    38.954980,-77.447956    10
+usage:   setLocation <lat>,<lng> <accuracy>
+example: setLocation 38.954980,-77.447956 10
 
 <lat> - Latitude
 <lng> - Longitude
 <accuracy> - Accuracy (in meters)
 ```
 
-#### setViewportSize
+#### `setViewportSize`
 Changes the size of the visible browser window so that the page viewport matches the given dimensions.  If you get black areas on your screen shots then the viewport is larger than the desktop.
 Browser Support: IE, Chrome, Firefox, Safari
 ```markup
-usage: setViewportSize	<width>    <height>
-example: setViewportSize    320    365
+usage:   setViewportSize <width> <height>
+example: setViewportSize 320 365
 
 <width> - Viewport Width
 <height> - Viewport Height
 ```
 
-#### sleep
+#### `sleep`
 Pauses the script operation for a given number of seconds.
 Browser Support: IE, Chrome, Firefox, Safari
 ```markup
-usage: sleep	<seconds to sleep>
-example: sleep	5
+usage:   sleep <seconds to sleep>
+example: sleep 5
 
 <seconds to sleep> - An integer indicating how long to sleep.  The allowable range is 1-30.
 ```
 
+## Debugging
```

Comment:
Adding a separate “Debugging” section because people may be searching for this.

Reply:
Good call.